db/migrate/20200813191955_create_posts.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePosts < ActiveRecord::Migration[6.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
+
+      t.timestamps
```

Comment:
# [__OPTIONAL__]
### Please follow The Odin Project naming convention on your code, since it will facilitate our job
> :text = :body